src/index.ts

```diff
@@ -706,6 +731,29 @@ export interface AcrolinxPlugin {
   showServerSelector?(): void;
 
   log?(logEntry: LogEntry): void;
+
+    /**
+   * Notifies the AcrolinxPlugin that a reuse prefix search has finished.
+   * @param reuseSearchResult  The reuse prefix search result.
+   */
+  onReusePrefixSearchResult?(reuseSearchResult: ReuseSearchResult): void;
+
+  /**
+  * Notifies the AcrolinxPlugin that a reuse prefix search has failed.
+  * @param prefix  The reuse prefix on which the search has failed.
+  */
+  onReusePrefixSearchError?(prefix: string): void;
+
+  /**
+  * the user has clicked on the button to open reuse panel. The AcrolinxPlugin is requested to open the reuse panel.
+  */
+  openReusePanel?(): void;
+
+  /**
+  * Notifies the AcrolinxPlugin that the user has changed the UI langauge in the sidebar.
+  * @param uiLanguage   The selected UI language
+  */
+  onUiLanguageChanged?(uiLanguage: UILanguage): void;
```

Comment:
this should be onU**I**LanguageChanged

Reply:
@JustinHorn Updated. Please make sure to also update it on your side